msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -550,6 +550,97 @@ Specify your Microsoft Azure Active Directory (Azure AD) App ID and Graph inform
 |`resource`|String|2048 characters|✔|Resource URL of the app for acquiring auth token for SSO.|
 |`applicationPermissions`|Array|Maximum 100 items|✔|Resource permissions for application.|
 
+## graphConnector
+
+**Optional**—object
+
+Specify the app's Graph connector configuration. If this is present then [webApplicationInfo.id](#webapplicationinfo) must also be specified.
+
+|Name| Type| Maximum size | Required | Description|
+|---|---|---|---|---|
+|`notificationUrl`|string|2048 characters|✔|The url where Graph-connector notifications for the application should be sent.|
+
+## showLoadingIndicator
+
+**Optional**—Boolean
+
+Indicates if or not to show the loading indicator when an app or tab is loading. Default is **false**.
+>[!NOTE]
+>If you select`showLoadingIndicator` as true in your app manifest, to load the page correctly, modify the content pages of your tabs and task modules as described in [Show a native loading indicator](../../tabs/how-to/create-tab-pages/content-page.md#show-a-native-loading-indicator) document.
+
+## isFullScreen
+
+ **Optional**—Boolean
+
+Indicate where a personal app is rendered with or without a tab header bar. Default is **false**.
```

Comment:
Should "Indicate where" be "Indicates when" or "Indicates whether" ?

Reply:
This one is tricky. Took a stab at clarifying